pdk/1.1/modules/ROOT/pages/policies-pdk-create-schema-definition.adoc

```diff
@@ -163,6 +163,48 @@ image:sensitive-data.png[The character inputs to the key parameter appear as bla
 
 Toggling *Show* reveals any new data currently being entered. However, you cannot reveal previously stored data. For example, if you configure the example *Key* parameter and apply the policy, you cannot view the parameter if you edit the policy's configuration in the future.
 
+=== Define Conditional Parameters
+
+Render policy configuration parameters dynamically by establishing conditions to make fields visible. In the following code example, the `encryptionKey` parameter only appears if the user selects `true` for the `encryptBody` parameter:
+
+[source,yaml]
+----
+properties:
+    encryptBody:
+        type: boolean
+        default: false
+    encryptionKey:
+        type: string
+        "@rendering":
+            "@visibleOn":
+                - property: encryptBody
+                  value: "true"
+----
+
+In the following code example, the `encryptionKey` parameter is initially visible and disappears if the user selects `false` for the for the `encryptBody` parameter:
+
+[source,yaml]
+----
+properties:
+    encryptBody:
+        type: boolean
+        default: true
+    encryptionKey:
+        type: string
+        "@rendering":
+            "@visibleOn":
+                - property: encryptBody
+                  value: "true"
+required:
+    - encryptBody
+----
+
+Conditional parameters have the following restrictions:
+
+* You can only define single condition per parameter.
+* Required properties that are conditionally rendered must have a default value defined.
```

Comment:
Maybe mention here that in the example, if `encryptionKey` is required, then it should have a default value.